### PR TITLE
CEFP-40 front end added

### PR DESCRIPTION
**Pull request by zoaa3054**

### CODE REVIEW

File: frontend/src/Components/SignupBox.jsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                                picture: user.picture
+                            })
+                        })
+                        .then(Response=>Response.status==201 ? navigate('/homepage', 
+                            {state: {user: {
+                                                username: user.name,
+                                                email: user.email,
+                                                password: '',
+                                                phone: '',
+                                                country: '',
```
Comment (alihassann191) [inline frontend/src/Components/SignupBox.jsx:58]: delete phone, country, city, address (unnecessary)

File: frontend/src/Components/SignupBox.jsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                                                city: '',
+                                                address: '',
+                                                gender: '',
+                                                picture: user.picture
```
Comment (alihassann191) [inline frontend/src/Components/SignupBox.jsx:62]: make picture field empty

File: frontend/src/Components/SignupBox.jsx (hunk 1 above)
Comment (zoaa3054) [reply frontend/src/Components/SignupBox.jsx:58]: We need them for ordering a product

File: frontend/src/Components/SignupBox.jsx (hunk 2 above)
Comment (zoaa3054) [reply frontend/src/Components/SignupBox.jsx:62]: Ok, but you will have to save the photo's path in the local folder in the server

File: frontend/src/Components/LoginBox.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          {theme: "outline", size: "larg"}
+        );
+    
+        google.accounts.id.prompt();
+      }, []);
+
+    const login = async (e)=>{
+        e.preventDefault()
+        try{
+            const userFetched = await fetch(`http://localhost:8081/users?email=${email}`);
```
Comment (alihassann191) [inline frontend/src/Components/LoginBox.jsx:58]: send the email and password in one request 
the expected API is "users/login"

**Pull request merged**